portal/src/main/webapp/js/src/load-frontend.js

```diff
@@ -56,11 +56,21 @@ window.loadReactApp = function(config) {
         });
 
     } else {
+        if (window.frontendUrl === undefined) {
+            if (console.error) {
+                console.error('ERROR: No frontend URL defined, should at least be empty string');
+            } else {
+                console.log('ERROR: No frontend URL defined, should at least be empty string');
+            }
+            // even though this should never happen, probably better to include
```

Comment:
maybe we should make it fail loudly though.  so devs will notice.  by falling back, they could think it's working